scripts/shelley-testnet.sh

```diff
@@ -54,8 +54,8 @@ tmux select-pane -t 4
 tmux send-keys "cd '${PWD}'; ${CMD} trace-acceptor $(acceptorargs)" C-m
 sleep 2
 tmux select-pane -t 1
-tmux send-keys "cd '${PWD}'; ${CMD} exe:cardano-node $(nodeargs 0 "${ALGO} $(echo -n ${EXTRA})") " C-m
+tmux send-keys "cd '${PWD}'; ${CMD} exe:cardano-node mock-protocol $(nodeargs 0 "${ALGO} $(echo -n ${EXTRA})") " C-m
 tmux select-pane -t 2
-tmux send-keys "cd '${PWD}'; ${CMD} exe:cardano-node $(nodeargs 1 "${ALGO} $(echo -n ${EXTRA})") " C-m
+tmux send-keys "cd '${PWD}'; ${CMD} exe:cardano-node mock-protocol $(nodeargs 1 "${ALGO} $(echo -n ${EXTRA})") " C-m
 tmux select-pane -t 3
-tmux send-keys "cd '${PWD}'; ${CMD} exe:cardano-node $(nodeargs 2 "${ALGO} $(echo -n ${EXTRA})") " C-m
+tmux send-keys "cd '${PWD}'; ${CMD} exe:cardano-node mock-protocol $(nodeargs 2 "${ALGO} $(echo -n ${EXTRA})") " C-m
```

Comment:
Was it using one of the test protocols before? Which one? How was that specified before?

Reply:
I made an error here, these are using `log-config-*.yaml` files which use `RealPBFT`. I associated mock with test, I'll fix it.